playground/react/App.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import Button from 'devextreme/renovation/button';
+
+const App = () => {
+    return <Button
+        text={'Click Me!'}
+        onClick={() => alert('Clicked')}
+        icon={'download'}
+        iconPosition={'right'}
+    ></Button>;
+};
```

Comment:
It would be more 'reactive' and better for our programmers. Or what about adding the `tslint` file here?

```jsx
const App = () => {
    return (
      <Button
        text="Click Me!"
        onClick={() => alert('Clicked')}
        icon="download"
        iconPosition="right"
      />
  )
};
```